docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -218,38 +177,19 @@ Explicitly instruct NetworkManager to never interact with your DNS configuration
   service NetworkManager restart
 
 RHEL 7.x
-^^^^^^^^
+++++++++
 
 NOTE: These are extra steps are required for RHEL 7 systems only, excluding derivatives such as CentOS or Scientific Linux.
 
 RedHat Enterprise Linux users need to take an additional setup step. If you are not using the RHN Subscription Management from RedHat you need to enable the optional channel by running the following as root:
 
   subscription-manager repos --enable rhel-7-server-optional-rpms
 
-Debian
-^^^^^^
-
-All the PacketFence dependencies are available through the official repositories.
-
-Software Download
-~~~~~~~~~~~~~~~~~
-
-PacketFence provides a RPM repository for RHEL / CentOS instead of a single RPM file.
-
-For Debian, PacketFence also provides package repositories.
-
-These repositories contain all required dependencies to install PacketFence. This provides numerous advantages:
-
-[options="compact"]
-* easy installation
-* everything is packaged as RPM/deb (no more CPAN hassle)
```

Comment:
I'd remove this line here, its been years we stopped using CPAN to install PacketFence